jacodb-analysis/src/main/kotlin/org/jacodb/analysis/codegen/Generator.kt

```diff
@@ -158,4 +163,81 @@ fun main(args: Array<String>) {
     }
 
     codeRepresentation.dumpTo(projectPath)
+    gradlewAssemble(targetLanguage, projectPath)
+}
+
+private fun gradlewAssemble(targetLanguage: TargetLanguage, projectPath: Path) {
+    val zipName = targetLanguage.projectZipInResourcesName()
+    val workingDir = File(projectPath.toFile(), zipName.substring(0, zipName.length - 4))
```

Comment:
I believe that you here want to find file name without file extension. There is builtin function to obtain such name in Path or File classes